minifi-toolkit/minifi-toolkit-assembly/NOTICE

```diff
@@ -0,0 +1,16 @@
+Apache NiFi
+Copyright 2014-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+===========================================
+Apache Software License v2
+===========================================
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Apache NiFi
```

Comment:
I took that from [the minifi assembly notice](https://github.com/apache/nifi-minifi/blob/a1d2fd3fe6b0b2cdd50394e2451aa10d5c9c81e9/minifi-assembly/NOTICE#L13).  Should I update that one too or address in separate jira?